plugins/cloudtrail/pkg/cloudtrail/aws.go

```diff
@@ -48,6 +50,10 @@ func (p *PluginConfigAWS) ConfigAWS() (aws.Config, error) {
 		opts = append(opts, config.WithSharedConfigProfile(p.Profile))
 	}
 
+	if len(p.Region) > 0 {
```

Comment:
it's a string, I don't understand why you check the length and not just the emptiness `if p.Region != ""` (same question for `p.Config)`